README.md

```diff
@@ -1,3 +1,49 @@
 # curriculum-cr-training-mobile-menu
 
 
+
+# My projects and about section 
+
+> This is a mobile menu project whose purpose is to do some practice of code review.
+
+## Built With
+
+- HTML 
+- CSS
+- JS 
+
+## Getting Started
+To get a local copy, just follow the steps That I am going to mention bellow:
+
+### Prerequisites
+
+### Setup
+To get this project in your local machine, follow these steps:
+- Open terminal 
+- clone the repository :https://github.com/ishimwezachee/curriculum-cr-training-mobile-menu.git
+- Cd in the project folder
+- run the project in your browser
+
+### install
+ - run npm install to install the required linters
+
+ ### tests
+ - run the project in the browsers 
+ - use google tool to inspect and check in mobile version.
+
+## Authors
+
+
```

Comment:
- [ ] you should consider adding your social media links into the authors section so that people who need more about your project can reach out to you .doing this will also make your readme professional hence the project

Reply:
Hello @mugishasam123, Thank you for your feedback. I am going to do my best to implement them :monocle_face: